app/assets/stylesheets/application.scss

```diff
@@ -13,3 +13,6 @@
  *= require_tree .
  *= require_self
  */
+@import "bootstrap";
+
+$favorite-color: 69BD83;
```

Comment:
The color is not coming out because you need this symbol, " # "
So it will look like this #69BD83;